src/components/AccountModal.vue

```diff
@@ -45,6 +45,27 @@ export default {
         {name: 'composition 4'}
       ]
     }
+  },
+  // declare all methods here
+  methods: {
+    logMe () {
+      console.log('Test Log from the methods declaration')
+    }
+  },
+  // this runs once per construction of this modal
+  created () {
+    // this.$services.compositions.create({
```

Comment:
remove comments

Reply:
Keeping this comment as it's a placeholder for what we need to do later once we can build compositions.